be/src/exec/aggregation_node.cpp

```diff
@@ -45,9 +45,9 @@
 
 namespace starrocks {
 
-// TODO: pass in maximum size; enforce by setting limit in mempool
-// TODO: have a Status ExecNode::init(const TPlanNode&) member function
-// that does initialization outside of c'tor, so we can indicate errors
+// Our new vectorized query executor is more powerful and stable than old query executor,
+// The executor query executor related codes could be deleted safely.
+// TODO: Remove old query executor related codes before 2021-09-30
```

Comment:
```suggestion
// TODO: unused code, will be deleted soon.
```

Reply:
The comments content we have discussed, we need more info in comments.